### PR TITLE
Add sass

### PR DESCRIPTION
**Pull request by ackers93**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: react-app/package.json [hunk 1]
```diff
@@ -1,53 +1,54 @@
...
+		"redux-thunk": "^2.3.0",
+		"typeface-montserrat": "0.0.75",
+		"typeface-roboto": "0.0.75"
+	},
+	"scripts": {
+		"start": "react-scripts start",
+		"build": "react-scripts build",
+		"test": "react-scripts test",
+		"eject": "react-scripts eject",
+		"scss": "sass --watch 'src/SCSS/main.scss' 'src/CSS/index.css'"
```
Comment (keveightysev) [inline react-app/package.json]: Please remove this script and install `node-sass`

File: react-app/src/index.jsx [hunk 2]
```diff
@@ -9,16 +9,17 @@ import thunk from 'redux-thunk';
 import logger from 'redux-logger';
 
 import './reset.css';
+import './CSS/index.css';
```
Comment (keveightysev) [inline react-app/src/index.jsx]: Once `node-sass` is install, you can just import './SCSS/main.scss'

File: react-app/package.json (hunk 1 above)
Comment (JossWritesCode) [reply react-app/package.json]: done

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
```
[the remark this answers is not in the document]
Comment (JossWritesCode) [reply .vscode/settings.json]: done

**Comment by JossWritesCode** [comment]
> Please:
> 
> * Delete the `package.json`, `yarn.lock`, `/node_modules`, and `/.vscode` from root folder
> * Install `node-sass` inside `/react-app`

Done. My apologies.

**Pull request merged**